v2/yarpcthrift/thriftrw-plugin-yarpc2/fx.go

```diff
@@ -151,7 +151,7 @@ type ServerResult struct {
 // 	)
 func Server(opts ...<$yarpcthrift>.RegisterOption) interface{} {
 	return func(p ServerParams) ServerResult {
-		procedures := <$server>.New(p.Handler, opts...)
+		procedures, _ := <$yarpc>.EncodingToTransportProcedures(<$server>.New(p.Handler, opts...))
```

Comment:
Given that this is v2 and is also in the generated code, we should update the returned function's signature to return an error and handle this.